crates/uv-python/src/discovery.rs

```diff
@@ -2360,6 +2457,12 @@ impl FromStr for VersionRequest {
     type Err = Error;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
+        // Stripping the 't' suffix produces awkward error messages if the user tries a version
+        // like "latest". HACK: If the version is all letters, don't even try to parse it further.
+        if s.as_bytes().iter().all(u8::is_ascii_alphabetic) {
```

Comment:
no need to go down to bytes, we can handle this on the char-level:
```suggestion
        if s.chars().all(char::is_alphabetic) {
```